### DISCUSSION TITLE
Jenkins Job build output to loki

### DISCUSSION THREAD

**Discussion by yzaltz** [oblador/loki#425] [Q&A]
Hi all,
New in Loki.
I'm interested in loading Jenkins Jobs output to Loki - what is the best way to do that ?

**Comment by cemtopkaya** [comment]
This could help you I think. 
https://grafana.com/docs/opentelemetry/collector/send-logs-to-loki/

There will be log files like `%JENKINS_HOME%/jobs/[JOB_NAME]/[BUILD_NUMBER like 73]/log` and you can get access with `http://localhost:8080/job/test_job/73/consoleText` url. If you give them config file like this:

```
receivers:
  filelog:
    include: [/usr/share/jenkins/jobs/**/log]
```

**Comment by techeverri** [comment]
Hey @yzaltz 👋

This is the repo for Loki the visual testing tool for Storybook.
- https://loki.js.org/

But you seem to be asking about Loki the log aggregation system from Grafana.
- https://grafana.com/oss/loki/

We can't really help you with Grafana here. I'd recommend you to open an issue in their repo
- https://github.com/grafana/loki

Happy coding 👍
